source/objects/mooringClass.m

```diff
@@ -42,10 +42,9 @@
         name                    = 'NOT DEFINED'                            % (`string`) Name of the mooring. Default = ``'NOT DEFINED'``
     end
 
-    properties (SetAccess = 'public', GetAccess = 'public') %internal
+    properties (SetAccess = 'private', GetAccess = 'public') %internal
         moorDyn                 = 0                                        % (`integer`) Flag to indicate a MoorDyn block, 0 or 1. Default = ``0``
         number                  = []                                       % (`integer`) Mooring number. Default = ``'NOT DEFINED'``        
-        orientation             = []                                            % (`float 1 x 6`) Initial 6DOF location. Default = ``[0 0 0 0 0 0]``        
     end

```

Comment:
are you sure this is unused?

Reply:
Yes this is never called in the mooring library or anywhere else in the mooring class. We probably should allow users to define the mooring orientation, but right now it's not possible. I thought to leave this for a larger discussion of how we define initial positions